Libraries/react-native/react-native.js

```diff
@@ -10,14 +10,18 @@
  */
 'use strict';
 
+var React = require('React');
+
 // Export React, plus some native additions.
 //
 // The use of Object.create/assign is to work around a Flow bug (#6560135).
 // Once that is fixed, change this back to
 //
-//   var ReactNative = {...require('React'), /* additions */}
+//   var ReactNative = {...React, /* additions */}
 //
-var ReactNative = Object.assign(Object.create(require('React')), {
+var ReactNative = Object.assign(Object.create(React), {
+  React,
```

Comment:
console.log should handle cycles anyway so you can debug cyclic objects. That said I think there isn't a cycle for ReactNative since the Object.create(React) is not the React object, so assigning React to the clone doesn't introduce a cycle.